scanner/src/lib/dataproducts/MobileWorkMachine_StraddleCarrier_OperationsData_v0.1.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  /*
+        Docs:
+
+        https://definitions.sandbox.ioxio-dataspace.com/definitions/DigitalProductPassport/MobileWorkMachine/StraddleCarrier/OperationsData_v0.1
+
+        https://github.com/ioxio-dataspace/sandbox-definitions/blob/main/src/DigitalProductPassport/MobileWorkMachine/StraddleCarrier/OperationsData_v0.1.py
+
+       https://gateway.sandbox.ioxio-dataspace.com/docs#/Data%20Products/DigitalProductPassport_MobileWorkMachine_StraddleCarrier_OperationsData_v0_1_DigitalProductPassport_MobileWorkMachine_StraddleCarrier_OperationsData_v0_1_post
```

Comment:
Indentation is off for the last entry compared to the above ones. Seems like these comments are in any case not uniform in these dataproducts/*.svelete files when it comes to indentation but would be kind of nice if they at least within each file would be.

Reply:
```suggestion
        https://gateway.sandbox.ioxio-dataspace.com/docs#/Data%20Products/DigitalProductPassport_MobileWorkMachine_StraddleCarrier_OperationsData_v0_1_DigitalProductPassport_MobileWorkMachine_StraddleCarrier_OperationsData_v0_1_post
```